server/src/main/resources/reference.conf

```diff
@@ -2,6 +2,13 @@ application {
   name: "Unnamed Tribe Service"
   domains: ["localhost"]
   modules: []
+  # Additional HTTPS security headers.  You can add, modify, remove as needed to enhance
+  # the security profile of our Kotlin based applications.
+  httpsHeaders: ["X-Content-Type-Options: nosniff",
```

Comment:
how do HTTP (not S) requests deal with these being set on an unencrypted connection (ie ENV=dev)?  are they just ignored, or are they functional over HTTP as well?  if they work there too maybe call this something like `securityHeaders` instead?  If they don't should they be disabled for dev (or only enabled for staging/prod) by default?

Reply:
They only work in staging and prod, not in ENV=dev.  Jersey ignores them when running in HTTP (not S).  I can move them to env specific config for ENV=staging or ENV=prod.